src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/AssetDefinitions/AssetDefinition.cs

```diff
@@ -0,0 +1,23 @@
+﻿using EasyAbp.BookingService.AssetCategories;
+using EasyAbp.BookingService.AssetSchedules;
+using JetBrains.Annotations;
+
+namespace EasyAbp.BookingService.AssetDefinitions;
+
+public class AssetDefinition
+{
+    [NotNull]
+    public string Name { get; }
+    
+    /// <summary>
+    /// This property determines whether assets can be occupied by default when there is no schedule created.
+    /// The SchedulePolicy of <see cref="AssetCategory"/> is preferred.
+    /// </summary>
+    public virtual AssetSchedulePolicy DefaultSchedulePolicy { get; }
+
```

Comment:
Feature suggestion: **Advance Booking Days**. Users should be able to configure the days before assets become available. For example, a hotel room at 10th July can be available when 1st July.
The property `AdvanceBookingDays` in `AssetDefinition` determines the days before this asset became available by default.

Reply:
Thanks. I will do this change and use the name `DaysInAdvance` (AdvanceBookingDays).

Reply:
maybe `AvailableDaysInAdvance` is better?

Reply:
I think "Available" is not needed, `DaysInAdvance` is clear and enough, or do you have any other better name?

Reply:
please consider that  `Asset` takes some days to prepare